src/components/postElements/body/view/postBodyView.js

```diff
@@ -127,23 +128,73 @@ const PostBody = ({
     }
   };
 
-  const _saveImage = uri => {
-    CameraRoll.saveToCameraRoll(uri)
-      .then(() => {
-        Alert.alert(
-          intl.formatMessage({ id: 'alert.success' }),
-          intl.formatMessage({ id: 'post.image_saved' }),
-          [{ text: 'OK' }],
-          { cancelable: false },
-        );
+  const checkAndroidPermission = async () => {
+    try {
+      const permission = PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE;
+      await PermissionsAndroid.request(permission);
+      Promise.resolve();
+    } catch (error) {
+      Promise.reject(error);
+    }
+  };
+
+  const _downloadImage = async uri => {
+    return RNFetchBlob.config({
+      fileCache: true,
+      appendExt: 'jpg',
```

Comment:
Do we have proper error handling if image meta type is different, like .gif or .png or .svg, etc.?

Reply:
i tried with gif image it is working. Actually i added this config for an android bug